src/pages/Epics/Epics.jsx

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class Epics extends Component {
+  state = {  }
```

Comment:
Can we initialize state in the constructor for all of these Components?

Reply:
I don't believe we have to actually initialize state in the constructor. For instance if the parent component passes in some props, in the state object we can reference this.props blah blah blah, so we can leave it like this for now.

Reply:
Okay.  I guess this is okay